bukkit/src/main/java/net/william278/huskhomes/hook/RedisEconomyHook.java

```diff
@@ -0,0 +1,25 @@
+package net.william278.huskhomes.hook;
+
+import dev.unnm3d.rediseconomy.api.RedisEconomyAPI;
+import net.william278.huskhomes.HuskHomes;
+import net.william278.huskhomes.HuskHomesInitializationException;
+import org.jetbrains.annotations.NotNull;
+
+public class RedisEconomyHook extends VaultEconomyHook {
+
+
+    public RedisEconomyHook(@NotNull HuskHomes implementor) {
+        super(implementor);
+    }
+
+
+    @Override
+    public boolean initialize() throws HuskHomesInitializationException {
+        RedisEconomyAPI api=RedisEconomyAPI.getAPI();
+        if(api!=null){
+            economy=api.getCurrencyByName(plugin.getSettings().currencyName);
```

Comment:
Style: Add spaces between operators (=, !=) etc -- see other classes for how I prefer code to be formatted if possible :)

Reply:
Done